distributed/cli/tests/test_dask_submit.py

```diff
@@ -0,0 +1,23 @@
+from __future__ import print_function, division, absolute_import
+from distributed.utils_test import valid_python_script, popen, invalid_python_script
+
+def test_dask_submit_cli(valid_python_script, invalid_python_script):
+    with popen(['dask-scheduler', '--port', '8989']) as sched:
+        with popen(['dask-submit', '127.0.0.1:8788', str(valid_python_script)]) as submit:
+            for x in range(5):
+                line = submit.stdout.readline()
+                if b'hello world!\\n' in line:
+                    break
+
+        with popen(['dask-submit', '127.0.0.1:8788', str(invalid_python_script)]) as submit:
+            for x in range(5):
+                line = submit.stdout.readline()
+                if b'Traceback' in line:
+                    break
+
+
+def test_submit_a_distributed_script(executor_contract_script):
```

Comment:
@mrocklin this test currently fails and i am not sure why. A trivial python script works fine but if i try to connect to scheduler via Executor, it never successfully establishes a connection. 

```
assert 'scheduler="127.0.0.1:8786" processes=0 cores=0' in [b'b\'Traceback (most recent call last):\\n  File "/var/folders/p4/6kv0q3sn02b5h9w1myf7l52rrhtbc2/T/client-mjpqip6q/di...ine 297, in wait\\n    gotit = waiter.acquire(True, timeout)\\nKeyboardInterrupt\\n\'\n', b'', b'', b'', b'', b'', ...]
```
any ideas on what might be going on?